hadoop-ozone/ozonefs/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneClientAdapterImpl.java

```diff
@@ -210,13 +210,33 @@ protected void incrementCounter(Statistic objectsRead) {
   }
 
   @Override
-  public OzoneFSOutputStream createFile(String key, boolean overWrite,
+  public OzoneFSOutputStream createFile(String key,
+      short replication,
```

Comment:
Please try to put parameters in the same line. One line one parameter is not recommended. 
Same comment for all the following piece of code.